src/pre.txt

```diff
@@ -0,0 +1,123 @@
+/* headers */
+.headers{
+    position: relative;
+    display: grid;
+    place-items: center;
+    width:70%;
+    margin-left: 400px;
+    margin-top: -50px;
+}
+#logo{
+width: 160px;
+height: 80px;
+position: relative;
+margin-left: 10%;
+margin-top: 15px;
+
+}
+
+.headers nav a{
+    text-decoration: none;
+    color: black;
+    font-size: large;
+    margin-left: 40px;
+
+}
+.headers button{
+    width: 120px;
+    height: 40px;
+    border-radius: 25px;
+    background-color: rgb(3, 3, 92);
+    color: white;
+    text-decoration: none;
+    margin-left: 80%;
+    border: none;
+    font-size: 1rem;
+    margin-top: -30px;
+}
+
+/* end of nav bar */
+/* introduction  */
+.wel-content{
+    position: absolute;
+    width: 500px;
+    height: max-content;
+    margin-left:24%;
+    margin-top: 18%;
+}
+.her-intro{
+    width: 750px;
+    height: 750px;
+    margin-left: 54%;
+    margin-top: 4%;
+   
+}
+
+.wel-content h1{
+    font-size: 4.0rem;
+    font-weight: bold;    
+font-family: 'Roboto', sans-serif;
+
+}
+.wel-content p{
+font-size: 1.7rem;
+font-family: 'Roboto', sans-serif;
+}
+span{
+    font-size: 1.7rem;
+    font-family: 'Roboto', sans-serif;
+}
+.wel-content button{
+    color: white;
+    background-color: black;
+    border: none;
+    margin-left: 150px;
+    margin-top: -30px;
+    height: 30px;
+    border-radius:20px 20px 20px;
+    width: 200px;
+    display: flex;
+    flex-direction: row;
+    }
+.wel-content #potential{
+    font-size: 0.8rem;
+    font-weight: bolder;
+    margin-left: 30px;
+    margin-top: 7px;
+
+}   
+.wel-content .material-symbols-outlined{
+width: 25px;
+background-color: white;
+color: black;
+border-radius: 50%;
+margin-left: 5px;
+margin-top: 2px;
```

Comment:
same

Reply:
sr thats the pre text not html file...but i deleted it so that it doesnt cause any issue